src/Aspire.Hosting/Publishing/ManifestPublisher.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace Aspire.Hosting.Publishing;
 
-public class ManifestPublisher(ILogger<ManifestPublisher> logger,
+internal class ManifestPublisher(ILogger<ManifestPublisher> logger,
```

Comment:
Same here.

Reply:
I don't think this deserves a one-off breaking change notice. I don't expect users to have used this type in their code. We could roll it in with the other breaking change notices for the appmodel.

cc @mitchdenny